packages/testcontainers/src/chains/defid_container.ts

```diff
@@ -22,7 +22,7 @@ export interface StartOptions {
 export abstract class DeFiDContainer extends DockerContainer {
   /* eslint-disable @typescript-eslint/no-non-null-assertion, no-void */
   public static readonly PREFIX = 'defichain-testcontainers-'
-  public static readonly image = 'defi/defichain:1.6.4'
+  public static readonly image = 'defi/defichain:HEAD-75b1420'
```

Comment:
```suggestion
  public static readonly image = 'defi/defichain:1.6.4'
```

revert this

Reply:
thanks for spotting! that's true this PR doesn't actually need the new node version